wiki/Ranking_Criteria/en.md

```diff
@@ -268,6 +269,7 @@ Custom naming must follow a common theme or pattern related to the song and must
 
 ### Guidelines
 
+-   **There should be no active sprites and commands after the end of the song.** This is flexible up to a few extra seconds depending on the storyboard effect, however more than that should not be necessary. 
```

Comment:
imo would replace however with 'but' since however is usually followed by comma in this situation

Reply:
done